README.md

```diff
@@ -104,7 +100,7 @@ Once the server is running, you can make a POST request with:
     {
         "query": "9887103",
         "documents": ["9887103", "30049242", "22936248"],
```

Comment:
This still has a query id in the list of returned document ids

Reply:
isn't that the request? not the response? The request can be made with the query id present in the documents but the response doesn't have it. I think?